### ISSUE TITLE
Update to lizmap 3.2 - blank page on login because of missing trait 'Lizmap\Form\WidgetTrait'

### ISSUE THREAD

**Issue by aeduard** [3liz/lizmap-web-client#1010]
Hi,
I had the following configuration: Ubuntu 16, Qgis server 2.18, PostgreSQL + PostGIS, PHP7, Lizmap 3.1 master. I decided to update to Lizmap 3.2, all the rest remained the same. I have followed the update procedure from the official lizmap documentation, but without success.
Then I decided to delete and reinstall lizmap, I've followed the procedure from the documentation, but the same situation happens. Lizmap main page loads OK, but when trying to connect, a blank white page appears.
I did not modify anything from my previous apache settings.
Link to the site: https://46.101.157.220/lm/ (ignore warning, it is because of the Self Signed Certificate)
Any ideas what I did wrong?

**Comment by jancelin** [comment]
test with Raspberry Pi of lizmap 3.2.0 and same problem when I want to login

```
Notice: Undefined property: stdClass::$error_handling in /www/lib/jelix/core/jCoordinator.class.php on line 351

Error 500. A technical error has occured. Sorry for this trouble.
```
no problem with 3.2rc6

**Comment by laurentj** [comment]
Please tell me details of errors that are into lizmap/var/log/errors.log.

You can also configure php to display errors : `display_errors=on` into php.ini (/etc/php/7.0/fpm/php.ini or /etc/php/7.0/apache2/php.ini). Don't forget to restart apache or php7.0-fpm. (change to `display_errors=off`again when all will work well)

**Comment by aeduard** [comment]
Lizmap errors.log doesn't display anything, no errors appear there. Instead php displays this:
 Fatal error: Trait 'Lizmap\Form\WidgetTrait' not found in /var/www/lizmap-web-client-master/lizmap/plugins/formwidget/input_htmlbootstrap/input_htmlbootstrap.formwidget.php on line 14

**Comment by laurentj** [comment]
@aeduard thank you, so your error is related to issue #991 . Are you sure you downloaded Lizmap 3.2 and not master (3.3) ?

**Comment by aeduard** [comment]
I think I've downloaded both time master, meaning 3.3. It is possible to verify the installed version on my server? changelog-en.txt is very old.

Before installing the new version I've deleted the whole lizmap-web-client folder from /var/www/ so temp/lizmap was deleted also.

I will try to reinstall the proper 3.2 version.

**Comment by laurentj** [comment]
@jancelin I think this is an other bug (probably related to your docker build), even if the result is the same as the @aeduard 's issue. Please open a new bug.

**Comment by laurentj** [comment]
@jancelin I suspect that the update-config file does not do a good job. I opened an issue on your project and will propose a patch.

**Comment by aeduard** [comment]
I have reinstalled Lizmap, this time 3.2.0 for sure. Everything works fine. I think by mistake I've installed master version, which is 3.3. Thank you for your help! Issue can be closed.

**Comment by laurentj** [comment]
@aeduard ok thanks. The master branch should also work now, as I fixed #991
